docs/fundamentals.txt

```diff
@@ -0,0 +1,26 @@
+.. _laravel_fundamentals:
+
+============
+Fundamentals
+============
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: php framework, odm
+
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
+
+   /fundamentals/read-operations
+   /fundamentals/write-operations
+
+Learn how to perform the following tasks by using the {+odm-long+} in the
+following pages:
```

Comment:
S: this sentence is a little repetitive; you could reword to something like:
```suggestion
Learn how to use the {+odm-long+} to perform the following tasks:
```

Reply:
Thanks, I like this reworded version!